pkgs/development/tools/clickable/default.nix

```diff
@@ -7,17 +7,19 @@
 , jsonschema
 , argcomplete
 , pytestCheckHook
+, watchdog
```

Comment:
unused parameter, otherwise diff LGTM.

Reply:
Accidentally removed its' use when reverting formatting.

Added it back to propagatedBuildInputs.